app/lakes/import.tsx

```diff
@@ -37,19 +37,32 @@ export default function TabImport() {
         <LoadFilesInput onChange={onChange} />
         <footer>
           <p>
-            <b>Accepted formats:</b> .csv, .json, .parquet, .tsv,{" "}
-            <Link href="https://github.com/brimdata/zed/blob/main/docs/formats/zng.md">
-              .zng
+            <b>Accepted formats:</b>{" "}
+            <Link href="http://ndjson.org/">NDJSON</Link>,{" "}
+            <Link href="https://docs.zeek.org/en/current/log-formats.html#zeek-tsv-format-logs">
+              Zeek TSV
             </Link>
             ,{" "}
-            <Link href="https://github.com/brimdata/zed/blob/main/docs/formats/zson.md">
-              .zson
+            <Link href="https://github.com/brimdata/zed/blob/main/docs/formats/zng.md">
+              ZNG
             </Link>
             ,{" "}
-            <Link href="https://github.com/brimdata/zed/blob/main/docs/formats/zst.md">
-              .zst
+            <Link href="https://github.com/brimdata/zed/blob/main/docs/formats/zson.md">
+              ZSON
             </Link>
           </p>
+          See{" "}
+          <Link href="https://github.com/brimdata/brim/wiki/Importing-CSV-JSON-Parquet-and-ZST">
+            docs
+          </Link>{" "}
+          for Zed platform support of{" "}
```

Comment:
Nit: I think "support for" reads better than "support of".
```suggestion
          for Zed platform support for{" "}
```